### ISSUE TITLE
ember-cli-typescript doesn't work with ember-electron

### ISSUE THREAD

**Issue by gossi** [emberwatch/ember-cli-typescript#28]
I tried using typescript on top of `ember-electron`, yet after installing and running `ember electron` I got the following error:

```
$ ember electron
⠧ Building2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: Error watching file for changes: EMFILE
    at exports._errnoException (util.js:1050:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1376:11)
```

That said `ember serve` runs normal.

Cross-Ref: felixrieseberg/ember-electron#246

**Comment by chriskrycho** [comment]
Thanks for the report, @gossi! Looks like this is the same as the error reported at #13, so I'm going to mark this as a duplicate and close it. It'll be a bit before I have time to fix this (I didn't write this add-on originally and I'm still coming up to speed on it and fixing other stuff), but I'll update #13 as I get it sorted. I'll also leave a comment linking to that issue over on ember-electron. 👍

**Comment by chriskrycho** [comment]
Note: this issue was fixed in #30 and that fix is in the 0.4.0 release going out momentarily!
